recipes/libevent/all/patches/0004-cmake-fix-runtime-install.patch

```diff
@@ -0,0 +1,22 @@
+diff --git a/cmake/AddEventLibrary.cmake b/cmake/AddEventLibrary.cmake
+index 04f5837e..88a19ca9 100644
+--- a/cmake/AddEventLibrary.cmake
++++ b/cmake/AddEventLibrary.cmake
+@@ -42,7 +42,7 @@ macro(export_install_target TYPE LIB_NAME OUTER_INCLUDES)
+         install(TARGETS "${LIB_NAME}_${TYPE}"
+             LIBRARY DESTINATION "lib" COMPONENT lib
+             ARCHIVE DESTINATION "lib" COMPONENT lib
+-            RUNTIME DESTINATION "lib" COMPONENT lib
++            RUNTIME DESTINATION "bin" COMPONENT lib
```

Comment:
Is this fixing the location of the main dll or of some helper dll's?
Can this patch be sent upstream?

Reply:
Don't really know. It was introduced by your commit 71ef2e30b096a44bb8ef811de4b314ca6d8648dd . I think all DLLs should be packaged in one directory, not the only one.

Reply:
Haha, I think I was fixing the location of the main dll. I don't know of any other library.

Reply:
on other hand, I think we can leave everything as is, and just set `cpp_info.libdirs` and `cpp_info.bindirs`